website/README.md

```
@@ -0,0 +1,54 @@
+# Starlight Starter Kit: Basics
+
+[![Built with Starlight](https://astro.badg.es/v2/built-with-starlight/tiny.svg)](https://starlight.astro.build)
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)